scripts/cbmc-viewer/cbmc-viewer

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+
+import report
+
+if __name__ == "__main__":
```

Comment:
why isn't this just in report?

Reply:
Removed this wrapper script and renamed report.py -> cbmc-viewer.